AppCenterData/AppCenterData/Internal/Util/MSDocumentUtils.m

```diff
@@ -65,7 +65,7 @@ + (MSDocumentWrapper *)documentWrapperFromData:(nullable NSData *)data
     }
     MSLogError([MSData logTag], @"Error deserializing data: %@", [error description]);
     MSDataError *dataError = [[MSDataError alloc] initWithErrorCode:MSACDataErrorJSONSerializationFailed innerError:error message:nil];
-    return [[MSDocumentWrapper alloc] initWithError:dataError documentId:nil];
+    return [[MSDocumentWrapper alloc] initWithError:dataError documentId:nil partition:nil eTag:nil];
```

Comment:
We might actually know the documentId and partition here. You would need to pass them to this method but look at where `documentWrapperFromData: documentType:fromDeviceCache:` is called, they are accessible : https://github.com/microsoft/AppCenter-SDK-Apple/blob/99417bf1de17f5df50890eabe1988ff37279007a/AppCenterData/AppCenterData/MSData.m#L520